webserver/src/main/java/io/kestra/webserver/controllers/ExecutionController.java

```diff
@@ -429,7 +435,7 @@ private Execution webhook(
     public Execution trigger(
         @Parameter(description = "The flow namespace") @PathVariable String namespace,
         @Parameter(description = "The flow id") @PathVariable String id,
-        @Parameter(description = "The inputs") @Nullable @Body Map<String, String> inputs,
+        @Parameter(description = "The inputs and labels") @Nullable @Body Map<String, String> inputsAndLabels,
```

Comment:
I'm not fan of mixing inputs and labels inside the body.
I think label are better passed as a query string argument in the request, it's also more logical as it's meta data and not mandatory.

Reply:
can be passed inside the body, we are in a multipart request, so it's make sense I think

Reply:
You cannot have multipl `@Body` annotation so if you don't want to break compatibility you must keep the hack implemented in this PR which prefix all labels by `labels` to distinguish them from inputs.
I'm really not a fan of it, and may lead to funny bugs (think if someone add an input that starts with labels) and hard to maintain code.

Reply:
Yea, the prefix _definitely_ is a hacky solution :) Although, binding the label values from POST query params felt a tad weird too. I'll try to switch to the query params.

Reply:
You can use `io.micronaut.http.annotation.Part` I think that avoid the queryString

Reply:
I've used the query parameters.